user_data.sh

```diff
@@ -0,0 +1,15 @@
+
+##############
+# Install deps
+##############
```

Comment:
This preamble is disproportionately large to the other comments. I think you can drop the `#######`